scripts/reboot

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+function stop_sonic_services()
+{
+    echo "Stopping sonic services..."
+    systemctl stop swss
+    systemctl stop teamd
+    systemctl stop bgp
+    systemctl stop lldp
```

Comment:
why only four dockers?

Reply:
snmp docker?

Reply:
Added stop snmp service to the list.

Thanks,
Ying